### PR TITLE
Implement the administrative unit's sites page

### PR DESCRIPTION
**Pull request by Windvis**

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ Router.map(function () {
     this.route('person', { path: '/:id/' }, function () {});
   });
   this.route('administrative-units', function () {
-    this.route('administrative-unit', { path: '/:id/' }, function () {});
+    this.route('administrative-unit', { path: '/:id/' }, function () {
+      this.route('sites', { path: '/vestigingen' }, function () {});
```
[the remark this answers is not in the document]
Comment (Windvis) [reply app/router.js:16]: Ok, I'll do it in a follow up PR.

[the remark this answers is not in the document]
Comment (anitacaron) [reply app/router.js:16]: I'll change them in the backend.

[the remark this answers is not in the document]
Comment (Windvis) [reply app/router.js:16]: No I mean in the frontend only. The thing that the user sees in the URL will be translated, but the route itself should remain in English.

[the remark this answers is not in the document]
Comment (anitacaron) [reply app/router.js:16]: Ah, ok, I thought they would be the same.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**